uint32 seq
uint8 flag_trigger
time stamp
string data
